published/024-2019-jan-8.md

```diff
@@ -0,0 +1,49 @@
+
++ May: Textile introduces a personal IPFS running on your phone for true decentralized storage. (https://twitter.com/opencholmes/status/997198068918509569) and announced the first ever IPFSConf (https://twitter.com/IPFSbot/status/998964366077816832) and released js-ipfs 0.29.0 (https://twitter.com/daviddias/status/1000996706211725312) and IPFS Cluster 0.4.0-rc2 is out (https://twitter.com/hecturchi/status/1001074944481120256) with newly launched website https://twitter.com/IPFSbot/status/1001083641072705536
+
++ June: 
+
++ July: Released js-ipfs v0.30.0 (https://twitter.com/daviddias/status/1016364733107228673) and v0.31.0 (https://twitter.com/_alanshaw/status/1023505986747883521) and js-libp2p just got its first _official_ release (https://twitter.com/daviddias/status/1022824309537026048) and go-ipfs, v0.4.17 was released (https://twitter.com/Whyrusleeping/status/1023116433113247746) and released Awesome IPFS website (https://twitter.com/daviddias/status/1024281949416771584)
+
++ August: held Lab Day 2018 in SF and released IPLD site (https://twitter.com/olizilla/status/1024701525250195458) and release of  https://js.ipfs.io  (https://twitter.com/daviddias/status/1024715960622542848) and released IPFS Cluster 0.5.0 (https://twitter.com/hecturchi/status/1033074533551538176)
+
++ September: Cloudflare now runs an IPFS gateway! (https://twitter.com/IPFSbot/status/1041756483967307776) and IPFS Weekly Newsletter returns (https://blog.ipfs.io/45-ipfs-weekly-11/) and js-ipfs 0.32.0 released (https://twitter.com/_alanshaw/status/1039552739221614594) and ipld-explorer-cli 0.14 was released (https://twitter.com/IPFSbot/status/1044816962705076225)
+
++ October: IPFS Pinbot emerges on Twitter (https://twitter.com/ipfspin/status/1053376847596187648)
+
++ November: js-ipfs 0.33 released (https://twitter.com/_alanshaw/status/1058020797766397952) and go-ipfs 0.4.18 has been released! (https://twitter.com/momack28/status/1062111837066424320) and js-libp2p v0.24.0 is here (https://twitter.com/jacobheun/status/1063430636189437954)
+
++ December: The HTTP client libraries are being renamed! (https://blog.ipfs.io/58-http-client-rename/)
+
+# Events:
+
+January: Lisbon IPFS for 2018 is the year of the Distributed Web 🙌🏽 🌐 (https://www.meetup.com/lisbon-ipfs/events/246513999/ & see talks here: https://twitter.com/daviddias/status/953388140785418240)
+
+IoT Dev Fest: https://twitter.com/chrismatthieu/status/957350700303794176
+
+February: WebVR/AR III: Mixed Reality Networks (https://www.eventbrite.com/e/webvrar-iii-mixed-reality-networks-cc360-tickets-40763651178#)
+
+May: Baltimore/DC IPFS Meet-Up (https://gateway.ipfs.io/ipfs/QmccuDo37eDmmWBRjJcpLJXiADnmRy7f59RWkkJQU7Jm97/)
+
+August: IPFS descended upon DWebSummit (https://twitter.com/IPFSbot/status/1024724255043280896)
+
+October: First IPFS Berlin meetup (https://www.meetup.com/IPFS-Berlin/events/254816369/) and IPFS Show & Tell in London (https://www.meetup.com/london-ipfs/events/255386386/)
```

Comment:
@renrutnnej There's now a playlist of videos of the talks from the London that might be more interesting to link to than the meetup listing, unless @olizilla (who hosted the event) thinks otherwise. https://www.youtube.com/playlist?list=PLuhRWgmPaHtRdiy0HKNy4UZ4dKVUVL_KG